lib/active_admin/resource/menu.rb

```diff
@@ -26,7 +26,7 @@ def default_menu_options
         {
           id: resource_name.plural,
           label: proc{ resource.plural_resource_label },
-          url:   proc{ resource.route_collection_path(params) },
+          url:   proc{ (controller.config.relative_url_root || '') + resource.route_collection_path(params) },
```

Comment:
Line is too long. [110/80]<br>Unnecessary spacing detected.<br>Space missing to the left of {.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.